templates/sst/default/templates.yaml

```diff
@@ -0,0 +1,35 @@
+dataset: sst
+subset: default
+templates:
+  5119a0b5-5d82-4401-900a-7fafc1d48ff6: !Template
+    id: 5119a0b5-5d82-4401-900a-7fafc1d48ff6
+    jinja: 'How positive is the movie review below?
+
+      Give a score on a scale from 0 to 1.
+
+
+      {{sentence}} |||
+
+      {{label}}'
```

Comment:
I just realized that this label is a fraction. Can you convert it into an integer between 1 through 5